.github/workflows/docker.yml

```diff
@@ -59,7 +59,7 @@ jobs:
 
       - name: Merge images
         run: |
-          ./tools/merge-docker-images.sh $GITHUB_SHA
+          ./tools/merge-docker-images.sh clockworklabs/spacetimedb "commit-${GITHUB_SHA:0:7}" "${GITHUB_SHA:0:7}-full"
```

Comment:
the first param should match the old definition of `TAG` in the original script below

the second param should match the old definition of `FULL_TAG` in the original script below